ci/build_python.sh

```diff
@@ -7,9 +7,16 @@ source rapids-env-update
 
 rapids-print-env
 
+version=$(rapids-generate-version)
```

Comment:
What version will we get here? I'm assuming we get `23.12` or some variant of that, is that right? We'll probably need to get the UCX-Py version based on the RAPIDS version instead, perhaps as we do in the [release script](https://github.com/rapidsai/ucx-py/blob/5b6b80d5de289360fbbb78c362939e3b0c296e96/ci/release/update-version.sh#L27)?

Reply:
The generated version will be based on the tags in the current repo, so actually it will already be correct for ucx-py. For example, [the packages built on the last CI run were `ucx-py-0.35.00a9`](https://github.com/rapidsai/ucx-py/actions/runs/6540962417/job/17761814013?pr=997#step:7:1194).